starlite/testing/test_client.py

```diff
@@ -77,7 +77,7 @@ def __init__(
             session_config: Configuration for Session Middleware class to create raw session cookies for request to the
                 route handlers.
         """
-        self.session = SessionMiddleware(app=app, config=session_config) if session_config else None
+        self.session = CookieBackend(config=session_config) if session_config else None
```

Comment:
Perhaps rename the key from `self.session` to something else? If this is not actually session data. If it is session data, i think there might be a bug here.

Reply:
I left this as `self.session` because that's how it was previously implemented. 

I was planning to do a full proper integration of the new session stuff once your #664 is merged.